privaterelay/templates/newlanding/a/plans.html

```diff
@@ -13,16 +15,15 @@
                         <li>{% ftlmsg 'landing-pricing-free-feature-1' %}</li>
                         <li>{% ftlmsg 'landing-pricing-free-feature-2' %}</li>
                     </ul>
-                    <a class="mzp-c-button mzp-t-product mzp-t-secondary mzp-t-lg">
+                    <a class="mzp-c-button mzp-t-product mzp-t-secondary mzp-t-lg" href="{% provider_login_url 'fxa' process='login' %}">
                         {% ftlmsg 'landing-pricing-free-cta' %}
                     </a>
                 </div>
                 <!--Premium Option-->
                 <div class="c-option c-premium">
                     <img class="c-brand-title" src="{% static 'images/logos/logo-firefox-premium-relay.svg' %}">
                     <p class="c-price premium-price">
-                        <span class="c-price-number">$0.99</span>
-                        <span class="c-currency">{% ftlmsg 'landing-pricing-premium-price' monthly_price='USD' %}</span>
```

Comment:
This is still needed. This string includes the `USD /month`

Even if we forgo to "USD" part, I think the /month is still recommended. 

<img width="906" alt="image" src="https://user-images.githubusercontent.com/2692333/138576788-403efc7f-6531-4493-8512-67e41b28f497.png">